### DISCUSSION TITLE
The docs could be improved regarding changing theme

### DISCUSSION THREAD

**Discussion by KotlinIsland** [JanDeDobbeleer/oh-my-posh#2620] [Troubleshoot] [answered]
The [Customize page](https://ohmyposh.dev/docs/installation/customize)  says:

> To change to for example the [jandedobbeleer](https://ohmyposh.dev/docs/themes#jandedobbeleer) theme, alter the Oh My Posh `init` line to the following (assuming you downloaded the configuration file to your user's home folder):

Why does it mention a config file downloaded to your user's home folder and not the theme folder? This is very misleading IMO, I would think that downloading or ricing custom config files would be a more advanced pattern, where this part of the docs should be more targeted at the basic usage.

Just something simple like:
>To set a new theme you need to change the `--config` option of the `oh-my-posh init` call in your profile script to point to the desired theme, there are many installed by default and you can preview them with the `Get-PoshThemes` command. (see Themes for more information)

(ps the `Get-PoshThemes` command has inconsistent folder separators at the end.)
![image](https://user-images.githubusercontent.com/65446343/183234673-25aae659-95ac-47de-9512-2aaa7bb260fc.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
That environment variable isn't available everywhere. Only on Windows the installer is able to create that. That's why. There's no way to know how the user installed oh-my-posh.

**Reply by KotlinIsland** [reply]
Fair enough, but in general I found the instructions hard to follow to do something simple like change theme. I've updated the OP to remove that part.

**Reply by JanDeDobbeleer** [reply]
@KotlinIsland what would have helped you?

**Reply by KotlinIsland** [reply]
The way the docs are written (regarding a config file downloaded to user profile dir) led me to believe that is the way to set a custom downloaded theme, I didn't understand that there isn't a `Set-PoshTheme` command or anything. I updated the OP with an example of how it might be clearer.

**Reply by JanDeDobbeleer** [reply]
@KotlinIsland had a quick look and I really don't see a lot of difference with the current text:

> The standard initialization sets Oh My Posh' [default theme][default-theme]. This configuration is downloaded
and kept up-to-date with Oh My Posh' version every time the shell starts. To change/alter the configuration, you need to
add the `--config` command-line option to change the configuration to use a specific file.

> ### Get inspiration

> The best place to get started is by looking at the [themes][themes], as these are **predefined configurations** to use/alter.
To change to for example the [jandedobbeleer][jandedobbeleer] theme, alter the Oh My Posh `init` line to the following (assuming you
downloaded the configuration file to your user's home folder):

**Reply by KotlinIsland** [reply]
I think the main issue is that the example given is regarding downloading a custom config file to a custom folder. As a new user I have absolutely no intention of following this seemingly advanced pattern and instead all I want to do is switch to a standard built-in theme, with no knowledge what so ever about how oh-my-posh is set up and that that there is even a folder of built in themes etc. IMO the way it's currently layed out is misleading.

**Reply by JanDeDobbeleer** [reply]
@KotlinIsland I gave it a go, should be available soon.

**Reply by KotlinIsland** [reply]
Thanks a bunch, I think this looks really good!
